cart/src/components/Cart.js

```diff
@@ -0,0 +1,38 @@
+import "../styles/Cart.css";
+
+const Cart = ({ cartArray, totalPrice, handleIncDec, removeCart }) => {
+  const cards =
+    cartArray.length === 0 ? (
+      <h1 className="empty">Cart is Empty</h1>
+    ) : (
```

Comment:
for the case where cart is empty, it is better to add a separate conditional return instead of using this ternary ( makes the code less readable).

```tsx
if (!cartArray.length)
  return <h1 className="empty">Cart is Empty</h1>;

return (
  <div className="cart">.......</div>
);
```